src/diceBot/YankeeYogSothoth.rb

```diff
@@ -93,6 +95,122 @@ def getRandomSkillTableResult(_command)
     return output
   end
 
+  # 二つ名決定表
+  def get_nickname_table(_command)
+    result1, _ = roll(1,6)
```

Comment:
この部分のコードの書き方について、Rubocopから修正案が出されていました。
https://travis-ci.org/bcdice/BCDice/jobs/645595686#L586-L590

* `result1` の後の `_` を省略する。
* 引数の `,` の後に空白を入れる。

```suggestion
    result1, = roll(1, 6)
```

Reply:
それぞれ対応しました。